ivadomed/main.py

```diff
@@ -73,11 +75,11 @@ def get_parser():
 
 
 def create_path_model(context, model_params, ds_train, path_output, train_onehotencoder):
-    path_model = Path(path_output, context.get("model_name"))
+    path_model = Path(path_output, context[ConfigKW.MODEL_NAME])
     if not path_model.is_dir():
         logger.info(f'Creating model directory: {path_model}')
         path_model.mkdir(parents=True)
-        if 'film_layers' in model_params and any(model_params['film_layers']):
+        if ModelParamsKW.FILM_LAYERS in model_params and any(model_params[ModelParamsKW.FILM_LAYERS]):
             joblib.dump(train_onehotencoder, path_model.joinpath("one_hot_encoder.joblib"))
             if 'metadata_dict' in ds_train[0]['input_metadata'][0]:
                 metadata_dict = ds_train[0]['input_metadata'][0]['metadata_dict']
```

Comment:
We might want to start thinking about pulling these BidsDataset keywords out eventually as well in a future PR as well. This one is already getting incredibly large.